recipes/nibabel/run_test.py

```diff
@@ -0,0 +1,14 @@
+import nose
+import sys
+import re
+
+splatform = sys.platform
+if splatform.startswith('win32'):
+    # Installation of data files for script testing a bit rickety on Appveyor:
+    config = nose.config.Config(verbosity=2,
+                                exclude=[re.compile("test_scripts"),
+                                         re.compile("test_dft")])
+else:
+    config = nose.config.Config(verbosity=2)
```

Comment:
Looks like genuine failures that only happen with python 3.5 for some reason and at least the AppVeyor log now has the full output of nosetests ...